library/std/src/process.rs

```diff
@@ -2158,26 +2148,30 @@ impl Termination for ! {
 }
 
 #[stable(feature = "termination_trait_lib", since = "1.61.0")]
-impl<E: fmt::Debug> Termination for Result<!, E> {
+impl Termination for Infallible {
     fn report(self) -> ExitCode {
-        let Err(err) = self;
-        eprintln!("Error: {err:?}");
-        ExitCode::FAILURE
+        // Doesn't matter, this is unreachable
+        ExitCode::SUCCESS
```

Comment:
NB: this impl is completely new and so maybe wants to be since 1.63.0 but idk how this works with trait impls anymore